handbook/engineering/cloud/index.md

```diff
@@ -103,6 +103,14 @@ Before team syncs, teammates and stakeholders should write down under "Discussio
 
 By default, members of the team will provide a brief update about the goals they're working on (one update per iteration goal) — this will be a short version of the written update in the tracking issue.
 
+## Team norms
+
+### Code reviews
+
+- **Post-merge feedback:** It is important to make progress while getting feedback from other teammates in code reviews. On the one hand, the pull request author doesn't have to be blocked by all reviewers who the author intends to get feedback from; on the other hand, reviewers can still focus on their work on hands and leave feedback at their convenience. It is at pull request author's best judgement if a pull request should wait (for high-risk changes) or simply rely on post-merge feedback.
```

Comment:
```suggestion
- **Post-merge feedback:** It is important to make progress while getting feedback from other teammates in code reviews. On the one hand, the pull request author doesn't have to be blocked by all reviewers who the author intends to get feedback from; on the other hand, reviewers can still focus on their work on hands and leave feedback at their convenience. The pull request author should use their best judgement to decide if a pull request should wait (for high-risk changes) or simply rely on post-merge feedback.
```

Reply:
I'm not quite sure what you mean by this:

```on the other hand, reviewers can still focus on their work on hands and leave feedback at their convenience```

Reply:
> I'm not quite sure what you mean by this:
> 
> `on the other hand, reviewers can still focus on their work on hands and leave feedback at their convenience`

Hmm, I was trying to say that everyone could do the work without spending all the time reviewing PRs (and worse, those PRs are blocking by their inputs).

Reply:
Ok, that makes sense